ddtrace/contrib/internal/langchain/patch.py

```diff
@@ -867,19 +868,23 @@ async def traced_lcel_runnable_sequence_async(langchain, pin, func, instance, ar
 
 @with_traced_module
 def traced_similarity_search(langchain, pin, func, instance, args, kwargs):
```

Comment:
I spent a bit extra time to add the async versions of each raced method and as many tests as possible. It does inflate the test suite though since we don't have a way to execute an async version of a function without duplicating the code.

Reply:
yeah, that's the unfortunate nature of adding async wrappers in Python, it's really gonna be a lot of duplicated code for like 1 line of change 😢 there are definitely use cases where users are using the async versions of these methods though, so I'd say it's good to support it.